source/streaming-mode/streaming-read-config.txt

```diff
@@ -46,6 +46,26 @@ You can configure the following properties when reading data from MongoDB in str
    * - ``collection``
      - | **Required.**
        | The collection name configuration.
+       | You can specify multiple collections by separating each collection name
```

Comment:
S: Probably worth mentioning somewhere that you can only stream from multiple collections in a single database (if I'm reading the Jira ticket correctly)